### ISSUE TITLE
example from README does not work in osx

### ISSUE THREAD

**Issue by opichon** [nginx-proxy/nginx-proxy#515]
Running the docker-compose,.yml example in osx with docker for mac doesn't work.

After running `docker-compose up -d`,  the command `curl -H "Host: whoami.local" localhost` returns this response:

```
<!DOCTYPE html>
<html>
<head>
<title>Welcome to nginx!</title>
<style>
    body {
        width: 35em;
        margin: 0 auto;
        font-family: Tahoma, Verdana, Arial, sans-serif;
    }
</style>
</head>
<body>
<h1>Welcome to nginx!</h1>
<p>If you see this page, the nginx web server is successfully installed and
working. Further configuration is required.</p>

<p>For online documentation and support please refer to
<a href="http://nginx.org/">nginx.org</a>.<br/>
Commercial support is available at
<a href="http://nginx.com/">nginx.com</a>.</p>

<p><em>Thank you for using nginx.</em></p>
</body>
</html>
```

**Comment by wader** [comment]
Seems to work for me with docker for mac. Do you see any requests logs in `docker-compose logs`?

**Comment by opichon** [comment]
After restarting docker for mac, it now works. Sorry for the trouble.

**Comment by phelpsiv** [comment]
Getting the same issue on both OS X and Debian using both Latest and 0.5.0.

Here's the docker log:

```
nginx-proxy     | forego     | starting nginx.1 on port 5000
nginx-proxy     | forego     | starting dockergen.1 on port 5100
nginx-proxy     | dockergen.1 | 2016/12/11 09:39:53 Generated '/etc/nginx/conf.d/default.conf' from 7 containers
nginx-proxy     | dockergen.1 | 2016/12/11 09:39:53 Error running notify command: nginx -s reload, exit status 1
nginx-proxy     | dockergen.1 | 2016/12/11 09:39:53 Watching docker events
nginx-proxy     | dockergen.1 | 2016/12/11 09:39:53 Contents of /etc/nginx/conf.d/default.conf did not change. Skipping notification 'nginx -s reload'
nginx-proxy     | nginx.1    | 172.18.0.1 - - [11/Dec/2016:09:39:56 +0000] "GET / HTTP/1.1" 200 612 "-" "curl/7.50.1" "-"
```

**Comment by kylegoetz** [comment]
@phelpsiv the sample YML is missing one thing in the whoami definition:

    expose:
      - "8000"

Try adding this. I had the exact same issue. I added that line and it works now. I also just submitted a PR for the README to include this instruction.

**Comment by tkw1536** [comment]
Fixed in #1184
